ytmusicapi/mixins/library.py

```diff
@@ -267,6 +267,25 @@ def rate_song(self, videoId: str, rating: str = 'INDIFFERENT') -> Dict:
 
         return self._send_request(endpoint, body)
 
+    def set_album_save(self, audioPlaylistId: str, saved=True):
```

Comment:
This is identical to [`rate_playlist`](https://github.com/sigma67/ytmusicapi/blob/43d3c3bc0455b73d44235d9d7adf7ad13aea35cc/ytmusicapi/mixins/library.py#L283)

I think it would suffice to add a comment to that method to state that it works on albums as well (if even that).

It is documented clearly in the FAQ: https://ytmusicapi.readthedocs.io/en/stable/faq.html#how-do-i-add-a-song-album-artist-or-playlist-to-my-library

So I would suggest to remove this.